sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, endpoint, credential, **kwargs):
             }
         )
         http_logging_policy.allowed_query_params.update(
-            {"includeTextDetails", "locale", "language", "includeKeys", "op", "pages"}
+            {"includeTextDetails", "locale", "language", "includeKeys", "op", "pages", "reading_order"}
```

Comment:
looks like it should be camel cased according to api spec: https://westcentralus.dev.cognitive.microsoft.com/docs/services/form-recognizer-api-v2-1-preview-3/operations/AnalyzeLayoutAsync

Reply:
Ah got it, I put the kwarg name. Thanks for letting me know! :)